app/views/posts/show.html.erb

```diff
@@ -1,27 +1,45 @@
+<script>
+  $(function() {
+    $('.wmd-output').each(function(i) {
+      var converter = new Markdown.Converter();
+      Markdown.Extra.init(converter);
+      var content = $.trim($(this).text());
+      $(this).html(converter.makeHtml(content));
+    });
+  });
+</script>
+

```

Comment:
@risu-kun you should save the above script as a Gist for adding markdown functionality to views.